CHANGELOG.md

```diff
@@ -36,6 +36,10 @@ Due to the removal of legacy code, there are a few breaking changes in this new
 
 ### New Features
 
+* Added the `contains`, `replace`, `hasPrefix`, `hasSuffix`, `upperFirstLetter`, `lowerFirstLetter` filters for strings
```

Comment:
It seems that you missed the `CHANGELOG.md` change in the commit. It's not in it 😉